tests/core/test_opds2_import.py

```diff
@@ -80,6 +81,9 @@ def _get_work_by_identifier(works, identifier):
 class TestOPDS2Importer(OPDS2Test):
     MOBY_DICK_IDENTIFIER = "urn:isbn:978-3-16-148410-0"
     HUCKLEBERRY_FINN_IDENTIFIER = "http://example.org/huckleberry-finn"
+    PROQUEST_IDENTIFIER = (
```

Comment:
I know this is not a product of the current work, but there is a weird asymmetry here: `<title>_IDENTIFIER`, `<title(ish)>_IDENTIFIER`, `<identifier-type>_IDENTIFIER`. Because they're used both in the context of the titles and as examples of identifier types, it might be helpful to make that more clear through their names. Maybe: `MOBY_DICK_ISBN_IDENTIFIER`, `HUCK_FINN_URI_IDENTIFIER`, and `POSTMODERNISM_PROQUEST_IDENTIFIER`.

Reply:
Good call. Updated the names for these identifiers.